understanding/text-spacing.html

```diff
@@ -58,268 +58,199 @@
+                     <p><a href="#dfn-style-property">スタイルプロパティ</a>の影響を受けず、適応が見込まれないテキストの例は次の通りである:
                      </p>
                      
                      <ul>
                         
-                        <li>Video captions embedded directly into the video frames and not provided as an associated
-                           caption file
+                        <li>動画のフレームに直接埋め込まれ、関連付けられたキャプションファイルとしては提供されない、動画のキャプション
                         </li>
                         
-                        <li><a href="#dfn-image-of-text">Images of text</a></li>
+                        <li><a href="#dfn-image-of-text">文字画像</a></li>
                         
                      </ul>
                      
-                     <p>For this SC, <a href="https://html.spec.whatwg.org/multipage/canvas.html#the-canvas-element">canvas</a> implementations of text are considered to be <a href="#dfn-image-of-text">images of text</a>.
+                     <p>この達成基準では、<a href="https://html.spec.whatwg.org/multipage/canvas.html#the-canvas-element">canvas</a> でのテキストの実装は<a href="#dfn-image-of-text">文字画像</a>と見なされる。
                      </p>
                      	
                   </section>
                   
                   <section id="use-of-ellipses">
                      
-                     <h4>Use of ellipses</h4>
+                     <h4>省略記号の使用</h4>
                      
-                     <p>There may be regions of a page where text containers cannot expand due to design constraints
-                        (such as a maximum width for the left navigation or table column headers). A common
-                        convention if text exceeds its space is to replace truncated text with an ellipsis.
-                        Where ellipses appear as a result of modifying text style properties, the page can
-                        still meet the Text Spacing requirements, so long as the content is still available.
-                        For example:
+                     <p>デザイン上の制約 (左ナビゲーションやテーブルの列見出しの最大幅など) で、テキストコンテナを広げることができないページ領域があるかもしれない。テキストがその間隔を超えた場合の一般的な慣例は、切り詰められたテキストを省略記号で置き換えることである。テキストスタイルプロパティを変更した結果、省略記号が表示された場合でも、コンテンツが利用可能である限り、そのページはテキストの間隔の要件を満たすとすることができる。例えば:
                      </p>
                      
                      <ul>
                         
-                        <li>a mechanism is provided to reveal the truncated text on the page (for instance, the
-                           text appears on focus or on activation)
+                        <li>切り詰められたテキストを開示するメカニズムがページ上に用意されている (例えば、フォーカス時やアクティブ化されたときにテキストが表示される)。
                         </li>
                         
-                        <li>where the ellipsis is part of a section of 
-                           content which includes a link, the truncated text is revealed on the linked page
+                        <li>省略記号がリンクを含むコンテンツの一部である場合、切り詰められたテキストはリンク先のページで開示される。
```

Comment:
revealは単に「示す」という意味でよいかと思います。
```suggestion
                        <li>省略記号がリンクを含むコンテンツの一部である場合、切り詰められたテキストはリンク先のページで表示される。
```